src/components/GearItemSelect.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from "react";
+import { useGearList } from "redux/api";
+
+import { GearSummary } from "apiClient/gear";
+
+import { Select } from "./Select";
+import { useDebounce } from "./useDebounce";
+
+type Props = {
+  value: string | null;
+  onChange: (person: GearSummary | null | undefined) => void;
+  className?: string;
+  invalid?: boolean;
+};
+
+export function GearItemSelect({ value, onChange, className, invalid }: Props) {
+  const [query, setInput] = useState<string>("");
+  const { pending, fn: debouncedSetInput } = useDebounce(setInput, 250);
+  const { gearList, isFetching } = useGearList({
+    q: query,
```

Comment:
Any value in having `missing: false` and `broken: false` here too? I'm guessing NO in case there are some inconsistencies with what gear is available and what gear is actually in the office.

But maybe a good compromise is to show a warning / error when the gear item returned is missing or broken? That might make it easy for approvers to see what is happening with the gear.

Reply:
Just created a side issue for this.